libs/repositories/src/cowApi.ts

```diff
@@ -4,6 +4,6 @@ const COW_API_BASE_URL = process.env.COW_API_BASE_URL || 'https://api.cow.fi';
 
 import type { paths } from './gen/cow/cow-api-types';
 
-export const cowApiClient = createClient<paths>({
-  baseUrl: COW_API_BASE_URL,
+export const cowApiClientMainnet = createClient<paths>({
+  baseUrl: COW_API_BASE_URL + '/mainnet',
```

Comment:
Not for now but, once we do add the SDK, use the orderbook api instance instead?

Reply:
I'd love to make the SDK to export this client. I don't love the current implementation of the SDK